src/cephadm/cephadmlib/daemons/nvmeof.py

```diff
@@ -76,12 +76,24 @@ def _get_container_mounts(data_dir: str, log_dir: str) -> Dict[str, str]:
         mounts[log_dir] = '/var/log/ceph:z'
         return mounts
 
+    def _get_tls_cert_key_mounts(
```

Comment:
@adk3798: Aviv mentioned that they're also adding the `--add-host=<hostname>` to the podman run cmdline. This seems to be required to force a match with the name specified in the TLS certificate.